frontend/src/components/shared/Radio/Radio.tsx

```diff
@@ -105,7 +105,7 @@ class Radio extends React.PureComponent<Props, State> {
                   style: ({ $isFocused }: { $isFocused: boolean }) => ({
                     marginBottom: 0,
                     marginTop: 0,
-                    paddingRight: fontSizes.twoThirdSmDefault,
```

Comment:
I think it'd be preferable to

* keep using a const defined in `fontSizes` as opposed to specifying a magic number in pixels (a new value may need to be added if there isn't an existing one that works)
* specify the padding here in rem as opposed to pixels

Bear in mind that I'm very far from a frontend expert (especially when it comes to styling-related things), so my intuition here could very well be wrong 😅

Reply:
Agree with Vincent.